emcc.py

```diff
@@ -1231,6 +1231,12 @@ def check(input_file):
       shared.Settings.TEXTDECODER = 0
       options.js_libraries.append(shared.path_from_root('src', 'library_pthread.js'))
       newargs.append('-D__EMSCRIPTEN_PTHREADS__=1')
+      if shared.Settings.WASM_BACKEND:
+        newargs += ['-pthread', '-matomics']
```

Comment:
I think can can just use `-pthread` as it implies `-matomics`.  Right @tlively ?

Reply:
That's right.